oleggerasinovich/Titanic.ipynb

```diff
@@ -0,0 +1,1686 @@
+{
```

Comment:
<p>Here you can rename columns <code>Age</code>  and <code>Survived</code>  to be <code>old_cnt</code>  and <code>survived_cnt</code>  to reduce ambiguity. It is hard to understand what you mean from the first sight.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/u2-de-students-lab/titanic_exploration/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>